_posts/2019-01-17-image-service-deprecation.md

```diff
@@ -10,9 +10,9 @@ categories:
 tags: []
 ---
 
-The AGRC brown out of the mapserv image services was very successful. We want to thank you for your participation and apologize for any inconveniences. The few people that were affected and caught by surprise are now informed and on the migration path forward.
+The AGRC brownout of the mapserv image services was very successful. We want to thank you for your participation and apologize for any inconveniences. The few people that were affected and caught by surprise are now informed and are on the migration path forward.
```

Comment:
Fyi, we decided on "brownout" (one word) for the prev announcement article, so I'd recommend consistency with that. 
Also, I'm thinking maybe we should reference that previous article and the brownout a little more specifically to give more context? Like "The January 5, 2019, brownout of image services, as described in XX article..." you know? And then link to it in case people need more background info?